### PR TITLE
updated LBNL team members & bios

### PR DESCRIPTION
**Pull request by nlharris**
two of the photos of Jackson Lab people are broken.

not sure if the absolute URL will work for sierra's photo.

### CODE REVIEW

File: public/team.yaml [hunk 1]
```diff
@@ -114,36 +114,31 @@ institutions:
...
         bio: Seth is an occasional software engineer for the Monarch Initiative. He specializes in tools for science and education and has participated in projects in a broad range of fields, including Bioinformatics, Planetary Science, Geology, and Linguistics. His current focus is on capturing and making use of ontology and annotation data. His educational background is undergraduate degrees in Mathematics and Computer Science (UCSC).
-      - name: Eric Douglass
-        picture: img/team/EricDouglass1.jpg
-        title: Software Engineer
-        alumni: false
-        bio: Eric is a software engineer that works with Seth and Chris on data validation pipelines, Turtle and Ontologies in triplestores and other tools. He comes from a background originally in physics from Lewis and Clark college and then software development in industry, Eric got his first taste of ontologies at his first tech job in Portland.
-      - name: Nathan Dunn
-        picture: img/team/NathanDunn.jpg
-        title: Software Engineer
-        alumni: false
```
Comment (nicolevasilevsky) [inline public/team.yaml:131]: I think rather than deleting Eric, you just want to set alumni: true

File: public/team.yaml [hunk 2]
```diff
@@ -114,36 +114,31 @@ institutions:
...
+        picture: https://raw.githubusercontent.com/berkeleybop/berkeleybop.github.io/master/people/sierra-moxon/smoxon.png
+        title: Software developer
+        alumni: false
+        bio: Sierra is a data architect and software developer with experience in model organism knowledgebases, the Alliance of Genome Resources, Intermine and agricultural biotechnology.
       - name: Justin Reese
         picture: img/team/JustinReese.jpg
         title: Software developer
         alumni: false
-        bio: Justin is a a computational biologist with an interest in integrating bioinformatics data to facilitate discovery, especially with respect to human disease. He develops software for the Monarch Initiative and other projects.
+        bio: Justin I am a computational biologist with an interest in integrating bioinformatics data to facilitate discovery, especially with respect to human disease. He leads the KG-COVID-19 project.
```
Comment (nicolevasilevsky) [inline public/team.yaml:137]: do you want this to read: Justin I am...?

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]

File: public/team.yaml (hunk 2 above)
Comment (nlharris) [reply public/team.yaml:137]: oops, thanks for catching that (copy/paste error)

File: public/team.yaml (hunk 1 above)
Comment (nlharris) [reply public/team.yaml:131]: i wasn't sure he ever actually worked on monarch, but ok

Comment (nicolevasilevsky) [reply public/team.yaml:131]: Oh, I may be wrong then.

**Pull request merged**